### PR TITLE
Feature/split api into common and public

### PR DESCRIPTION
**Pull request by nk-coding**
also adds generated graphql api documentation

depends on #19 and #18

### CODE REVIEW

File: website/scripts/generate-graphql-schema.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+mkdir -p ./website/schemas
```
Comment (chriku) [inline website/scripts/generate-graphql-schema.sh:1]: add shebang and `set -e`

File: website/scripts/update-graphql-doc.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+cd website
```
Comment (chriku) [inline website/scripts/update-graphql-doc.sh:1]: add shebang and `set -e`

**Review by chriku — CHANGES_REQUESTED** [review]

File: website/scripts/generate-graphql-schema.sh (hunk 1 above)
Comment (nk-coding) [reply website/scripts/generate-graphql-schema.sh:4]: 462dd558c674242483abd8f8ef865aafe2576ed7

File: website/scripts/update-graphql-doc.sh (hunk 2 above)
Comment (nk-coding) [reply website/scripts/update-graphql-doc.sh:4]: 462dd558c674242483abd8f8ef865aafe2576ed7

**Review by chriku — APPROVED** [review]

**Review by ModellbahnFreak — APPROVED** [review]

**Pull request merged**